Model/AdyenInitPayments.php

```diff
@@ -200,6 +239,34 @@ protected function buildPaymentsRequest(Quote $quote, array $stateData): array
             'channel' => self::PAYMENT_CHANNEL_WEB
         ];
 
+        // Only add these parameters if the user is logged in
+        if ($isLoggedIn) {
+            $isRecurringEnabled = $this->vaultHelper->getPaymentMethodRecurringActive(
+                $paymentMethodCode,
+                $storeId
+            );
+
+            $recurringProcessingModel = $this->vaultHelper->getPaymentMethodRecurringProcessingModel(
```

Comment:
Maybe this can go inside the if statement 
if recurring is enabled then fetch the processing model